packages/root-cms/ui/pages/DocTranslationsPage/DocTranslationsPage.tsx

```diff
@@ -249,9 +277,32 @@ DocTranslationsPage.Table = (props: DocTranslationsPageTableProps) => {
       <table className="DocTranslationsPage__Table">
         <thead>
           <tr>
-            <th>source</th>
+            <th className="DocTranslationsPage__Table__sourceCellHeader">
+              <div className="DocTranslationsPage__Table__headerLabel">
+                source
+              </div>
+            </th>
             {props.locales.map((locale) => (
-              <th key={locale}>{locale}</th>
+              <th
+                key={locale}
+                onClick={() => props.onSelectLocale(locale)}
+                className="DocTranslationsPage__Table__localeHeader"
+              >
+                <Tooltip
+                  className="DocTranslationsPage__Table__headerTooltip"
+                  placement="start"
+                  withArrow
+                  label={
+                    props.locales.length === 1 && props.locales[0] === locale
```

Comment:
this logic doesn't seem correct for the case when multiple locales are in the query params.

Reply:
fixed